commit e5b7fda7d84e8c79309842acdf1f48d9225d2567
Author: Alexandre Belloni <alexandre.belloni@free-electrons.com>
Date:   Tue Oct 25 11:37:58 2016 +0200

    ARM: at91: define LPDDR types
    
    commit e3f0a4017c2143b4b813df6a93e8cf79e3f76936 upstream.
    
    The Atmel MPDDR controller support LPDDR2 and LPDDR3 memories, add their
    types.
    
    Signed-off-by: Alexandre Belloni <alexandre.belloni@free-electrons.com>
    Signed-off-by: Sebastian Reichel <sre@kernel.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1229_e5b7fda7d84e8c79309842acdf1f48d9225d2567.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1229_e5b7fda7d84e8c79309842acdf1f48d9225d2567.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
